ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -3757,10 +3757,13 @@ private static Path mvFile(HiveConf conf, FileSystem sourceFs, Path sourcePath,
     } else if (isSrcLocal) {
       destFs.copyFromLocalFile(sourcePath, destFilePath);
     } else {
-      FileUtils.copy(sourceFs, sourcePath, destFs, destFilePath,
+      if (!FileUtils.copy(sourceFs, sourcePath, destFs, destFilePath,
```

Comment:
in some cases delete file failure is handled and some cases not

Reply:
delete failure handling is not in scope of this ticket. If any issues are there, let's handle as separate jira.